apps/src/code-studio/components/progress/CourseProgress.jsx

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
 import Radium from 'radium';
 import { connect } from 'react-redux';
-import _ from 'lodash';
-import { stageShape } from './types';
-import CourseProgressRow from './CourseProgressRow';
+// TODO: Delete CourseProgressRow
```

Comment:
strange, caught that myself when i reviewed and thought I force pushed removing it :man_shrugging: